src/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -103,6 +105,7 @@ public void IntPtrFieldSetValue(int value)
 
         [Theory]
         [MemberData(nameof(Pointers))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Pointers through Invoke not implemented: https://github.com/dotnet/corert/issues/2113")]
```

Comment:
Not sure about trackign work necessary for UWP in the CoreRT repo (as well as potentially CoreFX and PN databases)

But if we do we should mark it as milestone 2.1. I need to get rights to that repo to do it...